Cargo.lock

```diff
@@ -8180,7 +8180,7 @@ dependencies = [
  "aho-corasick",
  "bstr 1.9.0",
  "log",
- "regex-automata 0.4.3",
+ "regex-automata 0.4.6",
```

Comment:
Do we need all these updates? Especially if end up using standard methods on paths?